doc/governance.rst

```diff
@@ -50,13 +50,26 @@ nominated, there will be a vote by the current core contributors. Voting on new
 members is one of the few activities that takes place on the project's private
 mailing list. While it is expected that most votes will be unanimous, a
 two-thirds majority of the cast votes is enough. The vote needs to be open for
-at least 1 week.
-
-Core contributors that have not contributed to the project, corresponding to
-their role, in the past 12 months will be asked if they want to become emeritus
-members and recant their rights until they become active again. The list of
-members, active and emeritus (with dates at which they became active) is public
-on the scikit-learn website.
+at least 2 weeks.
+
+If a core contributor becomes inactive in the project for a period of one year, they
+will be considered for removal from the core contributors. Before removal, inactive
```

Comment:
If we decide to enforce this rule more strictly that we did in the past, we should probably define what we mean by "inactive" (e.g. no review / comment / created PR / create issue on github or participation in any dev meeting for 12 consecutive months or discussions on the mailing lists / discord server).

And we should probably have a script query the GitHub API to check for that.

Reply:
I'm a bit uneasy about this: it has reminiscence of [Goodhart's law](https://en.wikipedia.org/wiki/Goodhart%27s_law) "When a measure becomes a target, it ceases to be useful", which point to a real antipattern in steering social dynamics.